flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/source/reader/external/FetchTask.java

```diff
@@ -44,6 +44,9 @@
     /** Returns the split that the task used. */
     Split getSplit();
 
+    /** Stops current task, most of the implementations don't need this. */
+    default void stop() {}
+
```

Comment:
I don't think it's appropriate to declare the stop method as default.
I'd rather to do nothing when overriding it.

Reply:
Addressed in https://github.com/ververica/flink-cdc-connectors/pull/1838/commits/7490d9b9785adb1db065b68cd2305458e46920aa